### PR TITLE
Minor consistency corrections

### PR DESCRIPTION
**Pull request by kusterjs**
corrected:
- country PK assigned twice
- FIR of FARB, TXKF, ESMP, ESNF
- Prefix of LOVV-N

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -2666,7 +2665,7 @@ ESMI|Sjobo Sovde|55.598333|13.677222||ESMM|0
 ESMJ|Kagerod|55.995|13.053333||ESMM|0
 ESMK|Kristianstad|55.920556|14.085278||ESMM|0
 ESML|Landskrona|55.944444|12.869444||ESMM|0
-ESMO|Oskarshamn|57.351944|16.498333||ESOS|0
+ESMO|Oskarshamn|57.351944|16.498333||ESMM|0
```
Comment (Adrian2k) [inline VATSpy.dat:2668]: Why has this been changed to ESMM? The airport belongs to ESOS.

File: VATSpy.dat [hunk 2]
```diff
@@ -12801,7 +12800,7 @@ TVSC|Canouan|12.699167|-61.3425||TTZP|0
 TVSM|Mustique|12.88519|-61.1871||TTZP|0
 TVSU|Union Is./Union. - Intl|12.598611|-61.414722|UNI|TTZP|0
 TVSV|Arnos Vale|13.144444|-61.213889||TTZP|0
-TXKF|L F Wade Intl|32.364042|-64.678703|BDA|BDA-TMA|0
+TXKF|L F Wade Intl|32.364042|-64.678703|BDA|KZNY|0
```
Comment (Adrian2k) [inline VATSpy.dat:12803]: Shouldn't this be KSNY-BDA?

image

**Review by Adrian2k — CHANGES_REQUESTED** [review]

File: VATSpy.dat (hunk 1 above)
Comment (kusterjs) [reply VATSpy.dat]: Sorry, got the wrong line, it's ESMP to be changed

File: VATSpy.dat (hunk 2 above)
Comment (kusterjs) [reply VATSpy.dat]: changed

**Review by prryplatypus — APPROVED** [review]

**Review by Adrian2k — APPROVED** [review]

**Pull request merged**